joomla/joomla-cms
# [com_templates] Add submenu

#### Summary of Changes

This PR adds a submenu to com_templates, like the one that exists for "Manage" and "Languages".
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/14246062/d324a2c2-fa5c-11e5-9c27-04769c6cdbcd.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/14245986/b528089a-fa5c-11e5-96d8-344a08423628.png)
#### Testing Instructions
1. Use 3.5.0 or latest staging
2. Apply patch
3. Go to `Extensions -> Templates` and check if the submenu exists.
